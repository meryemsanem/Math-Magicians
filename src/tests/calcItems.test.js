import React from 'react';
// import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Items from '../components/calcItems';

test('renders the Items component', () => {
  const handleClick = jest.fn();
  const number = 8;

  const componentTree = renderer.create(
    <Items number={number} handleClick={handleClick} />,
  );

  expect(componentTree.toJSON()).toMatchSnapshot();
});
