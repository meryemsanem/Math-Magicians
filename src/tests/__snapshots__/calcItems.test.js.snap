// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Items component 1`] = `
<button
  className="input-btn"
  onClick={[MockFunction]}
  type="button"
>
  8
</button>
`;
